emission/core/wrapper/tour_model.py

```diff
@@ -0,0 +1,9 @@
+import logging
+import emission.core.wrapper.wrapperbase as ecwb
+
+class TourModel(obecwb.WrapperBase):
+
+    props = {"user_id" : ecwb.WrapperBase.Access.WORM} # user_id of the E-Missions user the graph represnts 
```

Comment:
Doesn't the tour model need any other information? like the list of common trips and common places? How am I supposed to retrieve the list of common places and trips from the tour model? Or if the intention is to just retrieve them directly, then why do we need a tour model wrapper?

Adding the decorators will make it much more clear how we plan on stitching together the objects.